docs/02-requisitos/priorizacao/backlog.md

```diff
@@ -0,0 +1,46 @@
+|||**US11:** Eu, como usuário, gostaria de cadastrar minhas receitas por voz para conseguir utilizar o site de formas mais diversas.|**Would**|
+||**FE05 - Busca**|**US12:** Eu, como usuário, gostaria de buscar uma receita para obter informações acerca do preparo da mesma.|**Must**|
+|||**US13:** Eu, como usuário, gostaria de buscar uma receita no próprio site para ter um acesso a informações delas.|**Must**|
+|||**US14:** Eu, como usuário, gostaria de buscar receitas em minhas conta para ter maior facilidade em achar as suas informações.|**Should**|
+|||**US15:** Eu, como usuário, gostaria de adicionar ingredientes alternativos para as receitas para que as pessoas possam flexibilizar o seu preparo.|**Could**|
+||**FE06 - Visualização**|**US16:** Eu, como usuário, gostaria de visualizar receitas para que eu possa ter acesso às informações delas.|**Must**|
+|||**US17:** Eu, como usuário, gostaria de deixar minha opinião em receitas e avaliá-las para que outros usuários possam ver a qualidade da receita.|**Could**|
+|||**US18:** Eu, como usuário, gostaria de conseguir ver opções de conversão de medidas para melhor executar as receitas.|**Could**|
+|||**US19:** Eu, como usuário, gostaria de poder ver alternativas para ingredientes para poder flexibilizar o preparo das receitas.|**Could**|
+||**FE07 - Compartilhamento**|**US20:** Eu, como usuário, gostaria de colocar uma receita como pública para tê-la visível ao público.|**Should**|
+|||**US21:** Eu, como usuário, gostaria de colocar uma receita como privada para mantê-la visível somente à minha conta ou contas autorizadas.|**Should**|
+|||**US22:** Eu, como usuário, gostaria de exportar minhas receitas em formato PDF para poder compartilhá-las.|**Could**|
+||**FE08 - Categorização**|**US23:** Eu, como usuário, gostaria de atribuir categorias à receitas para organizar as receitas em minha conta.|**Should**|
+|**EP03 - Livro de receitas**|**FE09 - Criação**|**US24:** Eu, como usuário, gostaria de adicionar receitas em meus livros de receitas para ter como organizar minhas receitas como eu bem quiser.|**Must**|
+|||**US25:** Eu, como usuário, gostaria de deletar meus livros de receitas da minha conta para poder retirá-los do site.|**Must**|
+|||**US26:** Eu, como usuário, gostaria de editar os meus livros para poder atualizar o conteúdo deles.|**Must**|
+|||**US27:** Eu, como usuário, gostaria de deletar as receitas do meu livro de receitas para poder atualizar o conteúdo deles.|**Should**|
+||**FE10 - Busca**|**US28:** Eu, como usuário, gostaria de buscar um livro de receitas no próprio site para obter receitas dentro de um mesmo domínio.|**Should**|
+|||**US29:** Eu, como usuário, gostaria de buscar receitas em meus livros para ter maior facilidade em achar as minhas receitas.|**Should**|
+||**FE11 - Visualização**|**US30:** Eu, como usuário, gostaria de visualizar livros de receitas para que eu possa ter acesso às informações deles|**Could**|
```

Comment:
Eu acho que a US30 poderia ter prioridade maior que a US31, que poderia ser um Would

Reply:
Pode ser que a US30 ser prioridade Should, enquanto visto que tem menos prioridade do que a visualização da receita em si, e deixar a US31 como Could mesmo. Eu estou usando a nomenclatura do Would são histórias que serão as últimas a serem feitas (se forem feitas)

Reply:
É que comentários em livros de receitas não é essencial, já que é apenas uma categorização pessoal pro usuário. O tipo de comentário que é importante é na receita (US17). Por isso sugeri rebaixar a US31 ao invés de subir a US30

Reply:
Entendi. Vou fazer as mudanças então